cloud/terraform/aws/eks/eks.tf

```diff
@@ -150,10 +180,40 @@ resource "aws_iam_role_policy" "this" {
 	)
 	role=aws_iam_role.irsa.id
 }
+resource "aws_iam_policy" "s3_buckets_read_access_policy" {
+	name="${var.deployment_name}-s3_buckets_access_policy"
+	policy=jsonencode(
+		{
+			Statement=[
+				{
+					Action=[
+						"s3:GetObject",
+						"s3:ListBucket",
+					]
+					Effect="Allow"
+					Resource=[
+						"arn:${var.arn_partition}:s3:::${var.liferay_extensions_bucket_name_prefix}-s3-bucket-*",
+						"arn:${var.arn_partition}:s3:::${var.liferay_extensions_bucket_name_prefix}-s3-bucket-*/*"
+					]
+					Sid="AllowS3BucketOperations"
+				}
+			]
+			Version="2012-10-17"
+		}
+	)
+}
 resource "aws_iam_role_policy_attachment" "role_policy_attachment_ebs_csi_driver" {
 	policy_arn="arn:${var.arn_partition}:iam::aws:policy/service-role/AmazonEBSCSIDriverPolicy"
 	role=aws_iam_role.ebs_csi_driver.name
 }
+resource "aws_iam_role_policy_attachment" "role_policy_attachment_irsa" {
```

Comment:
verify this liferay service account actually needs this policy attachment, to Liferay pod, the extensions are just files, liferay is not calling s3 API directly